test/api/compilation.js

```diff
@@ -0,0 +1,270 @@
+'use strict';
+
+const expect = chai.expect;
+const assert = chai.assert;
+
+import * as utils from '../utils.js';
+
+describe('test Compilation', function() {
+  const nn = navigator.ml.getNeuralNetworkContext();
+  const builder = nn.createModelBuilder();
+  const desc = {type: 'float32', dimensions: [2, 2]};
+  const a = builder.input('a', desc);
```

Comment:
operands seem to contain builder, can it provide chained API?
i.e. a.matmul(b)

Reply:
Created https://github.com/webmachinelearning/webnn/issues/106 to follow-up in spec.